containers/hydra/messager.py

```diff
@@ -2,17 +2,19 @@
 #!nix-shell -i python3 -p "python3.withPackages(ps: [ ps.slackclient ])"
 #
 #
-
-
+# This Slack messager needs 1) Slack authorization app token 2) slack channel to be messaged in a configuration file (in this order)
+# When Slack app is being created, the authorization token is given by the system. Please see Slack info how to create Slack apps.
+#
+# DO NOT STORE SLACK CONFIGURATION FILE TO GITHUB EVER. IT INCLUDES ACCESST TOKEN. ADD IT MANUALLY TO /containers/hydra directory FOR DOCKER
```

Comment:
Update comment after correcting implementation. containers/hydra is not the right place.